lib/chef-dk/policyfile_compiler.rb

```diff
@@ -123,7 +126,11 @@ def install
     end
 
     def create_spec_for_cookbook(cookbook_name, version)
-      source_options = default_source.source_options_for(cookbook_name, version)
+      matching_source = default_source.find { |s|
```

Comment:
Could this ever be nil (say because default_source is the null source)?  Would you want to grab default_source.first in case that happens?